### DISCUSSION TITLE
Adaptive Cards in Outlook not showing

### DISCUSSION THREAD

**Discussion by MNa19** [microsoft/AdaptiveCards#8646] [Q&A]
Dear Team,

I have 3 Flows with the same JSON Code for Adaptive Cards. The difference is only the trigger (date). But only one card can shown in Outlook and for the other ones i get the following error: 

_Error happened on card extraction for <BE1P281MB1601563AF0F7CBC5BD5FB970B90CA@BE1P281MB1601.DEUP281.PROD.OUTLOOK.COM>_

All cards can shown in Teams but not in Outlook.

Can you help me?

Thanks!

**Comment by jSquaredSolutions** [comment]
I am having the exact same issue. I'm trying to send an adaptive card via "Send and Email(V2) action:

Compose: 

`{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.0",
    "body": [
        {
            "type": "TextBlock",
            "id": "0b3a097f-1f21-4b9a-d5cf-72c6b20eb7a2",
            "text": "This is a text block. Insert your text here.",
            "wrap": true
        }
    ],
    "padding": "None",
    "@@type": "AdaptiveCard",
    "@@context": "http://schema.org/extensions"
}`

![image](https://github.com/microsoft/AdaptiveCards/assets/20631631/dd2b9ccc-f496-4613-b102-80c668c1e694)

**Reply by MNa19** [reply]
Have you already completed the provider registration (https://outlook.office.com/connectors/oam/publish) and entered the provider ID in the JSON code (before the body)? Try "originator" instead of "id"

Example:
![Screenshot 2024-01-02 201254](https://github.com/microsoft/AdaptiveCards/assets/141626695/53830a50-a4e5-4484-86a9-f3b3d64a2dc9)
